pubspec.yaml

```diff
@@ -10,7 +10,7 @@ authors:
   - Trent Grover <trent.grover@workiva.com>
 homepage: https://github.com/Workiva/w_flux
 dependencies:
-  react: ">=0.5.0 <2.0.0"
+  react: ">=0.5.0 <3.0.0"
```

Comment:
So it is only 0.9.0 is only needed for the examples. In order to not be a breaking change release this could be left as `react: ">=0.5.0 <3.0.0"` and we could have a dev_dependency of `react:  ">=0.9.0 <3.0.0"`